samples/MvvmSample.Core/Models/Post.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace MvvmSampleUwp.Models
```

Comment:
Perhaps we should change the namespaces in the new Core project to just be MvvmSample instead of MvvmSampleUwp?